src/e2e/java/teammates/e2e/cases/lnp/BaseLNPTestCase.java

```diff
@@ -336,9 +381,11 @@ protected void runJmeter(boolean shouldCreateJmxFile) throws IOException {
     protected void deleteDataFiles() throws IOException {
         String pathToJsonFile = getPathToTestDataFile(getJsonDataPath());
         String pathToCsvFile = getPathToTestDataFile(getCsvConfigPath());
+        String pathToStatisticsJsonFile = getPathToStatisticsResultsFile();
```

Comment:
Let's not delete the file? As of now, the `.jtl` results file isn't deleted either

Reply:
ohh so we are planning to keep diff file results is it? If that is the case, I will have to ensure that an unique filename is used for each test

Reply:
For now, yes. Let's keep it consistent.

If you can't rename the default file from "statistics.json", you can rename the file after it is generated